test/unit/js/flicking.test.js

```diff
@@ -2061,37 +1782,37 @@ test("When intent to initialize with non-existent element, should throw error.",
 	assert.throws(function() {
 		// When
 		new eg.Flicking("#no-child");
-	}, function( err ) {
+	}, function() {
 		// Then
 		return true;
 	});
 });
 
-test("Custom event name with prefix: to handle jQuery plugin style", function () {
+QUnit.test("Custom event name with prefix: to handle jQuery plugin style", function(assert) {
 	// When
 	var events = [
 			"flicking:beforeFlickStart",
 			"flicking:flick",
 			"flicking:flickEnd",
 			"flicking:beforeRestore",
 			"flicking:restore"
-		],
-		eventFired = [],
-		handler = function(e) {
+		];
+	var eventFired = [];
+	var handler = function(e) {
 			eventFired.push(e.eventType);
 		};
 
-	this.inst = new eg.Flicking("#mflick1",{ circular: true }, "flicking:").on({
+	var inst = this.inst = new eg.Flicking("#mflick1",{ circular: true }, "flicking:").on({
```

Comment:
create() is attaching event as non jQuery's plugin style.
That TC is to check a event binding in different event names.